setup.py

```diff
@@ -68,6 +68,7 @@ def _discover_tests():
             'scipy>=0.17',
             'shapely',
             'simplejson',
+            'Pillow'
```

Comment:
Wouldn't it be better if we finally migrate this to its own requirements file? Probably not in this PR, but just to keep it in mind. :)